src/pages/resultsView/fusion/IFusionMapperProps.tsx

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) 2018 The Hyve B.V.
+ * This code is licensed under the GNU Affero General Public License (AGPL),
+ * version 3, or (at your option) any later version.
+ *
+ * This file is part of cBioPortal.
+ *
+ * cBioPortal is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ **/
+
+import { FusionMapperStore } from './FusionMapperStore';
+import { IFusionMapperConfig } from './IFusionMapperConfig';
+
+export interface IFusionMapperProps {
+    store: FusionMapperStore;
+    config: IFusionMapperConfig;
```

Comment:
Maybe this interface can go in the file FusionMapperStore.tsx, so we add one less file?

Reply:
No worries for more files, at the end it will be merged into single file. And it's a good practice to always separate interface as single file